schemas/org.openweathermap.snowplow/weather_stamp/jsonschema/1-0-0

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "org.openweathermap.snowplow",
+    "name" : "weather_stamp",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "type" : "object",
+  "properties" : {
+    "dt" : {
```

Comment:
Is dt an epoch datetime? I suggest we convert that into ISO8601 for better handling in JSON and Redshift